Effects/Compile/3DSwing.fx

```diff
@@ -0,0 +1,68 @@
+float rotation; // The rotation of the sword sprite.
```

Comment:
Lets try not to commit things in the compile folder. Perhaps we should gitignore this...

Reply:
I'll also gitignore LightsaberTrail.fx. I'm not sure about the xnb files there though